doc/workflow.md

```diff
@@ -0,0 +1,148 @@
+# Digitizing plots for echemdb
+
+This step by step tutorial explains how to digitize plots commonly found in electrochemical research papers, e.g., cyclic voltammograms, for the [echemdb/website](https://github.com/echemdb/website) project. In this example a cyclic voltammogram is digitized. A plot is provided in Figure 2a in the <a href="./files/mustermann_2021_svgdigitizer_1/mustermann_2021_svgdigitizer_1.pdf" download>publication example</a>.
```

Comment:
I would strongly recommend against inline HTML as this will likely cause trouble down the road (e.g., with styling or with any other processing by Sphinx.)

I'll have a look what can be done instead.